src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -537,6 +557,7 @@ void LogError(IRestResponse response)
         static bool HasSucceeded(IRestResponse response) => successCodes.Any(x => response != null && x == response.StatusCode && response.ErrorException == null);
 
         static IEnumerable<HttpStatusCode> successCodes = new[] { HttpStatusCode.OK, HttpStatusCode.Accepted, HttpStatusCode.NotModified, HttpStatusCode.NoContent };
+        static Regex linkExpression = new Regex(@"<([^>]+)>;\s?rel=""(\w+)"",?\s?", RegexOptions.Compiled);
```

Comment:
Does SC really include non-encoded commas in the URL's? They should be encoded to `%2C`, then it would be a simple comma split. And semi-colons should be encoded to `%3B` allowing a further semi-colon split.